packages/tupaia-web/src/features/DashboardItem/EnlargedDashboardItem.tsx

```diff
@@ -183,11 +187,14 @@ export const EnlargedDashboardItem = () => {
     currentDashboardItem?.entityHeader || activeDashboard?.entityName
   }`;
 
-  const { type } = currentDashboardItem || {};
+  const { type, viewType } = currentDashboardItem || {};
 
   return (
     <Modal isOpen onClose={handleCloseModal}>
-      <Wrapper $hasBigData={reportData?.data?.length > 20 || type === 'matrix'}>
+      <Wrapper
+        $hasBigData={reportData?.data?.length > 20 || type === 'matrix'}
+        $applyWidth={viewType !== 'dataDownload'}
+      >
```

Comment:
Taking another look at this and the logic doesn't seem quite right somehow. I think that when $applyWidth is false nothing else matters. Could this be simplified a bit? I haven't tested this but I think it makes sense.

```suggestion
      <Wrapper
        $hasBigData={viewType !== 'dataDownload' && (reportData?.data?.length > 20 || type === 'matrix')}
      >
```